scaffold/template/custom/ios/fastlane/Fastfile

```diff
@@ -48,6 +48,8 @@ platform(:ios) do
       itc_team_name: team_name
     )
 
+    sh("fastlane", "produce", "disable_services", "--game-center", "ios")
```

Comment:
You can run this using `produce()` instead of in another instance of fastlane. Docs: https://docs.fastlane.tools/actions/produce/#fastlane-integration

It would also be helpful to add a comment about why this is here.

Reply:
> One minor change.
> 
> Did you run the images through an optimizer like [ImageOptim](https://imageoptim.com/mac)? If not please do so, it makes them significantly smaller.

Nope, I did not run them through an optimizer. I did try compressed files earlier and Apple complained that they were not the right dimensions even though they were. I then used https://makeappicon.com/ to generate these on @tan-duong's recommendation.

I'll run these through ImageOptim and see if it makes a difference. Thanks for the tip!

Reply:
> You can run this using `produce()` instead of in another instance of fastlane. Docs: https://docs.fastlane.tools/actions/produce/#fastlane-integration
> 
> It would also be helpful to add a comment about why this is here.

Yes, I tried adding the `enable_services` key with `game_center` set to `off` within `produce` but that did not work. There are open issues around that as well cause its not consistent for some reason.

Absolutely, good call on adding comments. Will highlight that this is a temporary fix while we wait for Apple to resolve https://developer.apple.com/forums/thread/707506?page=1#716538022